zstreamz/clients.py

```diff
@@ -0,0 +1,90 @@
+from collections import Sequence
+from concurrent.futures import ThreadPoolExecutor
+from functools import wraps
+
+from distributed import default_client as dask_default_client
+from tornado import gen
+
+from .core import identity
+
+
+def result_maybe(future_maybe):
+    try:
```

Comment:
Use if statement, more stable. If we need we can have a canonical list of Future-likes